### PR TITLE
Add CITATION.cff

### PR DESCRIPTION
**Pull request by ilkilic**

### CODE REVIEW

File: AUTHORS.txt
Comment (darshanmandge) [inline AUTHORS.txt:1]: So, do we want to keep this list different from those in CITATION.cff?

**Review by AurelienJaquier — APPROVED** [review]

File: README.rst [hunk 1]
```diff
@@ -27,22 +27,11 @@ For a general overview and example of electrical model building, please refer to
...
-      author       = {Damart, Tanguy and Jaquier, Aurélien and Arnaudon, Alexis and Mandge, Darshan and Van Geit, Werner and Kilic, Ilkan},
-      title        = {BluePyEModel},
-      month        = aug,
-      year         = 2023,
-      publisher    = {Zenodo},
-      doi          = {8283490},
-      url          = {https://doi.org/10.5281/zenodo.8283490}
-    }
+How to cite
+===========
```
Comment (AurelienJaquier) [inline README.rst:31]: Maybe you could keep the simple dash for title here, no?

File: AUTHORS.txt
Comment (ilkilic) [reply AUTHORS.txt:1]: Thanks, updated in the last commit

File: README.rst (hunk 1 above)
Comment (ilkilic) [reply README.rst]: Thanks, fixed in the last commit

**Review by darshanmandge — APPROVED** [review]

**Pull request merged**